hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1510,12 +1510,20 @@ public void performTableServices(Option<String> inFlightInstantTimestamp, boolea
       allTableServicesExecutedSuccessfullyOrSkipped = false;
       throw e;
     } finally {
+      String tableName = writeClient.getConfig().getTableName();
```

Comment:
Should the table name be renamed to have a suffix of `_mdt` based on the data table name so that the metric does not clash with the data table metrics and is easier to interpret?

Reply:
Addressed

Reply:
Hi @yihua , I noticed that here the table name already has a suffix `_metadata` to differentiate from the data table. So I think the `_mdt` suffix may not be needed